Made glass doors connect with frame blocks

Mineshafter61 opened the pull request:
Fixed the long-standing bug that made glass doors not connect to the door frame blocks.

Mineshafter61 committed:
Updated repository

Mineshafter61 committed:
bumped version

Mineshafter61 committed:
fix glass doors.

Signed-off-by: Mineshafter61 <yuhang.061@gmail.com>

Mineshafter61 committed:
Merge remote-tracking branch 'origin/master'

Mineshafter61 committed:
idk

Pecacheu committed:
Refactor

- Move project folder to root
- Switch to IntelliJ

Pecacheu committed:
v3.3.0

- Updated to latest Paper/Spigot 1.18.1
- Corrected indentation
- Okay, at least a little bit. Why, why did I make it so bad?

Pecacheu committed:
v3.3.1

- Minor bug fixes
- Code improvements
- Fixed issue with new config not being created on first start
- Fixed bugs with config load
- Removed saveInterval parameter in config (now saves automatically when Elevators change and when plugin shuts down)
- Call signs now affect redstone levers around them regardless of NODOOR setting.
- Redstone now updates correctly

Pecacheu committed:
v.3.3.2

- Implement setDoorBlock glass pane bugfix by @Mineshafter61

Note: Forgot to mention, implemented some fixes by @WolvHaven in v3.3.1! Thanks for the forks guys

Pecacheu commented:
Heya, I implemented your fix and some of @WolvHaven's changes in version 3.3.2. That bug is finally fixed! (And idfk I'm not sure how to merge your fork across another fork or anything, I just mentioned you in the commit xd)
And yes, I'm actually not dead. Amazing, I know

Pecacheu committed:
v3.3.3

v3.3.3
- Fix bug #19 with adding new elevator columns on tall elevators
- Fix entities not moving quite in sync with moving floor
-  Fix bug #16
- Prevent players from leaving world/logging out to keep their flying status!
- Create new signs of matching type when origin sign is a different type besides oak wood
- Fix bug with glass pane door blocks not always being reset when they should be
- Right-click air now moves elevator (thanks to friend for suggestion)
- Optimize code lots

Pecacheu committed:
v3.3.4

- New error messages for elevator not found or user lacking permissions
- Fix call sign level detection when call sign slightly above/below elev sign (3 blocks below, 1 above)
- Fix call sign redstone activation when on different level than elev sign
- Simplify some API functions
- Fix plugin crash when re-creating elevator that was deleted before, but already has call signs on some floors (Whoops!)
- Fix long-standing bug where dirt block not correctly added below player when an upper floor destroyed, causing player to fall to their doom in Survival mode!
- Correctly remove invalid floors when they are too close (floors should always be at least 3 blocks apart in height! If we didn't enforce this, it would be possible to destroy your own elevator with the floor blocks, and crash the plugin!)
- Fix deprecation issues with sign text read/write
- LOTS more optimization of routines, event handler code is actually somewhat sensible now
- You can now rename a level without deleting it by creating a new sign on that floor and naming it something different, it will update that name on ALL elevator signs on that floor!
- Add elev list command to list all elevators.

Pecacheu committed:
Update README

Mineshafter61 committed:
Merge branch 'master' into pr/1

# Conflicts:
#	.gitignore
#	pom.xml
#	src/main/java/com/pecacheu/elevators/ChuList.java
#	src/main/resources/config.yml
#	src/main/resources/plugin.yml

Mineshafter61 committed:
yes
